scottbarnesg/smart-sec-cam
# Fix docker-compose build hanging with large video directory

Changes:

- Adds .dockerignore w/ entry to ignore video data dir

Testing:

- Create `data/videos` and put lots of data into it.
- Check out this branch and run `docker-compose build server`
- Verify the command does not hang

Fixes #20
